tests/linker/mac/link all/LinkAllTest.cs

```diff
@@ -28,9 +29,15 @@ static void Check (string calendarName, bool present)
 		public void Calendars ()
 		{
 			Check ("GregorianCalendar", true);
+#if __MACOS__ // I'm not sure if this is the expected behavior for macOS, or if it's a bug somewhere.
```

Comment:
It's unclear how dotnet uses ICU (internally os system) on macOS, c.c. @steveisok

Reply:
Filed to have a look at this later: https://github.com/xamarin/xamarin-macios/issues/12468